frontend/src/features/verifiable-fields/VerifiableFieldService.ts

```diff
@@ -54,13 +56,13 @@ export const triggerSendOtp = async ({
   fieldId: string
   /** The value of the verification field to verify. Usually an email or phone number */
   answer: string
-}): Promise<void> => {
+}): Promise<SendFormOtpResponseDto> => {
```

Comment:
Will this mean that if the new UI sends a request to the old server, the frontend will break? Since it the new UI will expect `{ otpPrefix: string }` but that won't be served by the old servers.

Reply:
Yeah. I assume that aside from the initial release window, our BE almost be on the latest version as opposed to the FE where user doesn't refresh.

Reply:
Maybe we'd want to include this as a note in the PR? Though the impact of this is probably low (unless we cut the release at the same time as a huge event involving OTPs).

Reply:
Ah, makes sense. Forgot that this is still released together.